romana-install/roles/romana/postinstall/kubernetes/master/templates/demo.sh

```diff
@@ -49,6 +49,13 @@ function get_pods() {
 function get_pod_ip() {
     kubectl ${2:+--namespace "$2"} get pod "$1" -o json | jq -r '.status.podIP'
 }
+function delete_tenant() {
```

Comment:
The `function` keyword isn't actually necessary. But the rest of them in the script use it, so OK.